src/extensions/extension-manager.ts

```diff
@@ -97,6 +97,15 @@ export class ExtensionManager {
     }
   }
 
+  getNpmPackageTarballUrl(packageName: string) {
+    try {
+      const command = [this.npmPath, "view", packageName, "dist.tarball", "--silent"];
+      return child_process.execSync(command.join(" "), { encoding: "utf8" }).trim();
```

Comment:
Currently doesn't work in production build.. Any ideas?

Reply:
Not sure why, but I don't think we want to block the application while running this command? Why not use .exec?

Reply:
Doesn't matter anymore, I'll remove auto-detection from NPM as we support only direct URLs.